src/test/unit/conftest.py

```diff
@@ -105,6 +116,8 @@ class WebInterfaceUnitTestConfig(BaseModel):
     database_mock_class: Type = CommonDatabaseMock
     #: A class mocking :py:class:`~intercom.front_end_binding.InterComFrontEndBinding`
     intercom_mock_class: Type[CommonIntercomMock] = CommonIntercomMock
+    #: A class mocking :py:class:`~intercom.front_end_binding.InterComFrontEndBinding`
+    redis_mock_class: Type[StatusInterfaceMock] = StatusInterfaceMock
```

Comment:
I don't like the name `redis_mock_class`. It isn't really mocking redis.
What about  `status_mock_class`?

Also the doc comment is a copy paste artifact and refers to the wrong class.

Reply:
fixed
for whatever reason I also accidentally named the class Rest... instead of Redis... -> fixed that also